lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -63,7 +66,7 @@
                                 screenReaderTitle: gettext('Reset your Password'),
                                 valueAttribute: 'password',
                                 emailAttribute: 'email',
-                                linkTitle: gettext('Reset Password'),
+                                linkTitle: gettext('Reset Your Password'),
                                 linkHref: fieldsData.password.url,
                                 helpMessage: gettext('When you click "Reset Password", a message will be sent to your email address. Click the link in the message to reset your password.')
```

Comment:
When you click "Reset Your Password", edX will send a message to the email address for your edX account. Click the link in the message to reset your password.